### PR TITLE
pages_controller_index

### PR DESCRIPTION
**Pull request by aearamos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,68 @@
...
+
+- Doesn't have any database support.
+- Should be designed to expose an interface that allows separate components to interact with it
+
+The API should support the multiple places booking will accept the number of places to book (up to 8 and) and the name of the user, then it will return the position of the places booked.
+
+The booking logic has the following rules:
+
+1) if the places can all fit inside a lane, they will be booked in the same row
+2) if the places don't fit inside a lane, they will be balance across rows
```
Comment (gerard-morera) [inline README.md]: Not sure if using the verb balance helps me to understand what it means. If you think is correct lets use `balanced`.

File: README.md [hunk 2]
```diff
@@ -2,36 +2,68 @@
...
+3) if the plane is full and the seats cannot be balanced across rows, they will booked across the aisle
```
Comment (gerard-morera) [inline README.md]: `be booked`.

File: README.md [hunk 3]
```diff
@@ -2,36 +2,68 @@
...
+4) The booking should start from the windows seat, when possible.
```
Comment (gerard-morera) [inline README.md]: window

File: README.md [hunk 4]
```diff
@@ -2,36 +2,68 @@
...
+booking.book('Marco', 4)
+booking.book('Gerard', 2)
 
-When the service receives a multiple seats request, it should chose the seats following the rules described below:
+booking.show
+```
 
-- If the number of person is inferior of the lane size, you should sit them all in line
-- [Optional] If the number of person is greater then the lane size, you should place the seat nearby on multiple lanes. (Ex. for a short range airplane if we book 4 places they can be set an B1, C1, B2, C2)
+Given an empty plane, here are some expected behavior for the booking.
```
Comment (gerard-morera) [inline README.md]: Given an empty plane, `this is` the expected behavior for the booking.

File: README.md [hunk 5]
```diff
@@ -2,36 +2,68 @@
...
+  * Gerard seats: 'E1', 'F1'; 
+```
 
 You should:
 
-- Provide a the source code, some description about the design decision and the instruction about how to execute the it.
+- Add information about your design decision.
 - Write production-ready code.
 - Design the code in order to be extensible.
+- Suggest possible enhancements
```
Comment (gerard-morera) [inline README.md]: Misses dot at the end.

File: README.md [hunk 6]
```diff
@@ -2,36 +2,68 @@
...
 |---------------|-------------|-------------|--------------------------|
 | :short_range  | 156         | 26          | A B C _ D E F            |
-| :long_range   | 360         | 36          | A B C _ D E F G _ H I J  |
 
 
-The task is to write the logic that describe the booking service.
+The service:
+
+- Doesn't have any database support.
+- Should be designed to expose an interface that allows separate components to interact with it
```
Comment (gerard-morera) [inline README.md]: Misses dot at the end.

File: README.md [hunk 7]
```diff
@@ -2,36 +2,68 @@
...
+
+The API should support the multiple places booking will accept the number of places to book (up to 8 and) and the name of the user, then it will return the position of the places booked.
+
+The booking logic has the following rules:
+
+1) if the places can all fit inside a lane, they will be booked in the same row
```
Comment (gerard-morera) [inline README.md]: Missing dot at the end os several points.

File: README.md (hunk 4 above)
Comment (mtanzi) [reply README.md]: `Those are the expected behaviors` - they are more then one

File: README.md (hunk 3 above)
Comment (mtanzi) [reply README.md]: `window seats`

File: README.md [hunk 8]
```diff
@@ -2,36 +2,68 @@
...
 |---------------|-------------|-------------|--------------------------|
 | :short_range  | 156         | 26          | A B C _ D E F            |
-| :long_range   | 360         | 36          | A B C _ D E F G _ H I J  |
 
 
-The task is to write the logic that describe the booking service.
+The service:
+
+- Doesn't have any database support.
+- Should be designed to expose an interface that allows separate components to interact with it.
```
Comment (gerard-morera) [inline README.md:15]: I think that we need to rephrase this sentence but maybe we can omit it. What do you think @Badiapp/backend

File: README.md [hunk 9]
```diff
@@ -2,36 +2,68 @@
...
+
+The API should support a multiple seats booking expecting the name of the passenger and the number of places to book (up to 8 and). As result it will return the positions of all the places booked.
```
Comment (gerard-morera) [inline README.md]: (up to 8 ~~and~~)

**Pull request closed without merging**